BlazarUI/app/scripts/components/shared/LazyRender.jsx

```diff
@@ -44,10 +44,6 @@ let LazyRender = React.createClass({
   },
 
   getHeight: function(numChildren, childHeight, maxHeight) {
-    let fullHeight = numChildren * childHeight;
-    if (fullHeight < maxHeight) {
-      return fullHeight;
-    }
     return maxHeight;
   },
 
```

Comment:
Yeah I made this change last minute to fix a bug. What do you want passed in as an optional prop?